index.html

```diff
@@ -16,7 +16,7 @@
 
 <body id="index-body">
     <div id="header-container">
```

Comment:
### USE GOOD HTML SEMENTICS

- Instead of Creating a separate div for "header-container", you can add HTML <header> .